.github/pull_request_template.md

```diff
@@ -21,19 +21,23 @@ SPDX-License-Identifier: Apache-2.0
 <!-- We must be able to understand the design of your change from this description. If we can't get a good idea of what the code will be doing from the description here, the pull request may be closed at the maintainers' discretion. -->
 <!-- Keep in mind that the maintainer reviewing this PR may not be familiar with or have worked with the code here recently, so please walk us through the concepts. -->
 
-### Benefits
-
-<!-- What benefits will be realized by the code change? -->
-
 ### Possible Drawbacks
 
 <!-- What are the possible side-effects or negative impacts of the code change? -->
 <!-- If no drawbacks, explicitly mention this (write None) -->
 
-### Usage Examples or Tests <!-- Optional -->
+## Checklist Before Opening a PR
+
+Before you open a Pull Request (PR), please make sure you've completed the following steps and confirm by answering 'Yes' to each item:
 
-<!-- Point reviewers to the test, code example or documentation which shows usage example of this feature -->
+1. **Code is formatted**: Have you run your code through clang-format to ensure it adheres to the project's coding standards? 
+<!-- Yes/No -->
```

Comment:
If you want the PR opener to leave Yes or No under each point, then I suggest substituting the template with the easier one for editing:
```suggestion
Yes No
```
Now the inapplicable option is getting removed with just double click + backspace.

Reply:
I hope it is not desired to actually type or leave `Yes` letters under each question. In case of `Yes` the reviewer just leaves anything untouched.

Thus, I suggest in case of any `No` (if so) add an instruction to explicitly type the corresponding `No` with the mandatory reasoning.

Reply:
May use YAML forms like https://github.com/libp2p/cpp-libp2p/pull/230?

Reply:
@turuslan I introduced these forms for issues, but unfortunately they are not available for PRs yet